homeassistant/components/advantage_air/climate.py

```diff
@@ -75,32 +76,62 @@ async def async_setup_entry(
             entities.append(AdvantageAirAC(instance, ac_key))
             for zone_key, zone in ac_device["zones"].items():
                 # Only add zone climate control when zone is in temperature control
-                if zone["type"] != 0:
+                if zone["type"] > 0:
                     entities.append(AdvantageAirZone(instance, ac_key, zone_key))
     async_add_entities(entities)
 
 
 class AdvantageAirAC(AdvantageAirAcEntity, ClimateEntity):
     """AdvantageAir AC unit."""
 
+    _attr_fan_modes = [FAN_LOW, FAN_MEDIUM, FAN_HIGH]
     _attr_temperature_unit = UnitOfTemperature.CELSIUS
     _attr_target_temperature_step = PRECISION_WHOLE
     _attr_max_temp = 32
     _attr_min_temp = 16
-    _attr_fan_modes = [FAN_AUTO, FAN_LOW, FAN_MEDIUM, FAN_HIGH]
-    _attr_hvac_modes = AC_HVAC_MODES
-    _attr_supported_features = (
-        ClimateEntityFeature.TARGET_TEMPERATURE | ClimateEntityFeature.FAN_MODE
-    )
 
     def __init__(self, instance: dict[str, Any], ac_key: str) -> None:
         """Initialize an AdvantageAir AC unit."""
         super().__init__(instance, ac_key)
-        if self._ac.get("myAutoModeEnabled"):
-            self._attr_hvac_modes = AC_HVAC_MODES + [HVACMode.AUTO]
+        self._attr_unique_id = f'{self.coordinator.data["system"]["rid"]}-{ac_key}'
```

Comment:
```suggestion
```

I _think_ this can be tripped as it looks like its already appended in `AdvantageAirAcEntity` and this would otherwise be a breaking change

Reply:
Oh very nice catch! I think the output shoudl be exactly the same but this line isnt required.

Reply:
Right. You were re-setting it not append :). Guess its time for more coffee on my part